modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -4920,6 +4920,35 @@ public void testGetObjectEntries() throws Exception {
 			).build());
 	}
 
+	@Test
+	public void testGetObjectEntriesTotalCountMatchesItemCount()
+		throws Exception {
+
+		// Company scope
+
+		ObjectEntry objectEntry = _addObjectEntry(
+			_objectDefinition2, Collections.emptyMap());
+
+		_defaultObjectEntryManager.deleteObjectEntry(
+			_simpleDTOConverterContext, _objectDefinition2,
+			objectEntry.getId());
+
+		_testGetObjectEntriesTotalCountMatchesItemCount(
+			_objectDefinition1, _group);
+
+		// Site scope
```

Comment:
You only need to implement for one type of scope, so use the company-scoped object like we have for the methods below:

`DefaultObjectEntryManagerImplTest#testMoveObjectEntryToTrash`
`DefaultObjectEntryManagerImplTest#testRestoreObjectEntryFromTrash`